tests/test_models.py

```diff
@@ -2015,7 +2014,7 @@ def test_fork_callback(self):
     @mock.patch('website.archiver.tasks.archive')
     def test_register_callback(self, mock_archive):
         registration = self.node.register_node(
-            None, self.consolidate_auth, '', '',
+            None, self.consolidate_auth, {}
```

Comment:
Dangit, right. It'll work for post requests with basic auth, but not for model methods that expect an  `Auth` instance.